app/src/main/java/org/gdg/frisbee/android/onboarding/AppInviteLinkGenerator.java

```diff
@@ -63,4 +78,30 @@ private String createDeepLink(String gplusId) {
             .toString();
     }
 
+    private void shareShortAppInviteLink(String longUrl) {
+        FirebaseDynamicLinksHub firebaseDynamicLinksHub = App.from(activity).getFirebaseDynamicLinksHub();
+        String requestString = "{\"longDynamicLink\": \"" + longUrl + "\",\"suffix\": {\"option\": \"SHORT\"}}";
+        RequestBody body = RequestBody.create(okhttp3.MediaType.parse("application/json; charset=utf-8"), requestString);
+        firebaseDynamicLinksHub.getShortenedUrl(BuildConfig.FIREBASE_WEB_API_KEY, body).enqueue(new Callback<FirebaseDynamicLinksResponse>() {
+            @Override
+            public void onSuccess(FirebaseDynamicLinksResponse response) {
+                createChooser(HttpUrl.parse(response.getShortLink()));
+                shareProgressDialog.dismiss();
```

Comment:
Hey! Sorry for the delayed response. I've been trying to get Screenhero to work on my system but couldn't. I even re-installed Windows to get it working, but in vain.

I can still contribute though. Let me know what you need me to do.